evalml/tests/utils_tests/test_gen_utils.py

```diff
@@ -215,17 +215,17 @@ def _check_equality(data, expected, check_index_type=True):
         pd.testing.assert_frame_equal(data, expected, check_index_type)
 
 
-@pytest.mark.parametrize("data,num_to_pad,expected",
-                         [(pd.Series([1, 2, 3]), 1, pd.Series([np.nan, 1, 2, 3])),
+@pytest.mark.parametrize("data, num_to_pad, expected",
```

Comment:
nit: I didn't realize you can add spaces here, but I'd personally prefer no spaces to keep the formatting consistent with the other tests we have in other files